server/routes.js

```diff
@@ -120,8 +121,8 @@ module.exports = (app) => {
   /**
    * Groups.
    */
+  app.post('/groups', ifParam('flow', 'github'), aN.authenticateAppByApiKey, aN.parseJwtNoExpiryCheck, aN.checkJwtExpiry, required('payload'), groups.createFromGithub); // Create a group from a github repo
   app.post('/groups', aN.authenticateUserByJwt(), required('group'), groups.create); // Create a group. Option `role` to assign the caller directly (default to null).
```

Comment:
For a bit more coherence (and to avoid having the need for line 124 to be before 125), `ifParam` could also be added on this route + add e.g `?flow=manual` in client call.